src/extractor/node_based_graph_factory.cpp

```diff
@@ -191,7 +212,11 @@ void NodeBasedGraphFactory::CompressAnnotationData()
     const auto old_size = annotation_data.size();
     // remove all remaining elements
     annotation_data.erase(new_end, annotation_data.end());
-    util::Log() << " graoh compression removed " << (old_size - annotation_data.size())
+
+    // reorder data in the same order
+    std::sort(annotation_data.begin(), annotation_data.end());
+
+    util::Log() << " graph compression removed " << (old_size - annotation_data.size())
```

Comment:
Please add here time output for the method scope. for `germany-latest` in release it costs 13.5 seconds, but interesting is timing for a planet-sized extraction.